ci/release_notes.md

```diff
@@ -0,0 +1,25 @@
+# Improvements
+
+- Genesis now supports the Genesis Index, a free service available
+  online at https://genesis.starkandwayne.com that tracks releases
+  and stemcells from bosh.io and Github.  Genesis now uses this to
+  supply version, tarball URL and SHA1 checksum information for
+  both releases and stemcells.
+
+  This makes it possible for newer BOSH CLIs to transparently
+  upload missing releases and stemcells to your BOSH director,
+  which in turn enables Concourse pipelines to easily roll
+  stemcell and release upgrades through unattended.
+
+  One side-effect of this change is that the "latest" version
+  keyword now no longer means "latest available on the BOSH
+  director", but instead "latest known to the index".
+
```

Comment:
@dennisjbell had similar concerns, although his complaint mostly centered around when we go to the index -- he advocated talking to the index at `genesis add release` time, where latest would transform to latest-at-that-time and you'd download the URL/SHA1 at that time and not every time you `make deploy`.

I'm open to this.

Thoughts?